content/en/docs/ops/configuration/traffic-management/dns-proxy/test.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1090,SC2154
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# @setup profile=none
+
+set -e
+set -u
+set -o pipefail
+
+source "tests/util/samples.sh"
```

Comment:
This line doesn't seem to be needed.

Reply:
now it is needed, as I am using sleep application

Reply:
I would suggest starting the sleep sample using `snip_dns_capture_in_action_2`, instead of the util function. Tests should try to use the commands that are in document snips whenever they can. Remember the primary purpose of a doc test is to run the commands that appear in the doc to make sure they work as documented.